docs/examples/basic_conceptual_guide/Part_3-model_analyzer/Readme.md

```diff
@@ -0,0 +1,108 @@
+
+![Triton Architecture](./img/arch.jpg)
+
+Once a query is sent from the client, Triton's handler enqueues it in the requested model's queue. Once one of the model instances is freed up, a dynamic batch of size corresponding to the preferred batch size is formed using the queries other incoming queries or queries which are already in the queue. This batch is then converted to the format required by the framework and sent to the Framework runtimes (PyTorch, TensorFlow, TensorRT, etc.). Post inference, these results are returned to the client.
+
+In this process, there are three major contributors of latency:
+* Network Latency
+* Inference Compute time
+* Latency caused due to wait time in the model's queue
+
+Minimizing **Network latency** is a case by case process. For instance, let's consider computer vision models. These models use image frames, videos and 3D data like point clouds which can be quite large in size, thus requiring higher bandwidth to move. Most images are saved in float32 format, which can be converted to float16 format. This may affect the dynamic range in the image, which may or may not affect model performance based preprocessing steps taken while training the model but can definitely reduce the latency as less data needs to be transmitted.
+
+Accelerating models to trim down the effective **compute time** is commonly achieved with a plethora of techniques like: optimizing the network graphs by fusing layers, reducing models precision, fusing kernels and more! This topic is covered in much more depth in Part 4 of this series.
+
+Latency in **queues** can primarily be addressed by adding more instances of the model. This may or may not require additional resources based on the GPU utilization of the current number of instances. This is the core utilization problem that needs to be solved specifically for each deployment environment. To streamline this experience, Triton Inference Server comes with Model Analyzer.
+
+Model Analyzer is a CLI tool to help with a better understanding of the compute and memory requirements of the Triton Inference Server models by sweeping through configurations settings and generating reports summarizing performance.
+
+With Model Analyzer users can: 
+* Run customizable configuration sweeps can be done to identify the best possible configuration for the expected workload and hardware. 
+* Summarize findings about latency, throughput, GPU resource utilizations, power draw and more, with detailed reports, metrics and graphs. These reports help compare performance across different configurations of setup.
+* Tailor model deployments to cater to user's Quality of Service requirements like specific p99 latency limits, GPU memory utilization and more!
```

Comment:
and more is really just minimum throughput, so it's better just to list this one as well